package-lock.json

```diff
@@ -6661,20 +6656,20 @@
       }
     },
     "node_modules/docusaurus-lunr-search": {
-      "version": "2.3.2",
-      "resolved": "https://registry.npmjs.org/docusaurus-lunr-search/-/docusaurus-lunr-search-2.3.2.tgz",
-      "integrity": "sha512-Ngvm2kXwliWThqAThXI1912rOKHlFL7BjIc+OVNUfzkjpk5ar4TFEh+EUaaMOLw4V0BBko3CW0Ym7prqqm3jLQ==",
+      "version": "3.3.2",
```

Comment:
It would be better to use the same version, there are two different versions 3.3.1 and 3.3.2 :sweat_smile: